homeassistant/helpers/entityfilter.py

```diff
@@ -0,0 +1,74 @@
+"""Helper class to implement include/exclude of entities and domains."""
+
+from homeassistant.const import (
+    CONF_ENTITIES, CONF_DOMAINS)
+from homeassistant.core import split_entity_id
+
+
+class EntityFilter():
+    """Class that implements an include/exclude filter for entities/domains."""
+
+    def __init__(self, include_dict, exclude_dict):
+        """Set up the filter."""
+        self.include_e = include_dict.get(CONF_ENTITIES, [])
```

Comment:
Let's not add knowledge about the configuration to the constructor. Instead pass include_entities, include_domains, exclude_entities and exclude_domains in.